queries.sql

```diff
@@ -15,4 +16,43 @@ SELECT * From animals Where neutered = true ;
 /* The 7th qeury*/
 SELECT * From animals Where name not in ('Gabumon'); 
 /* The 8th query*/
-SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
\ No newline at end of file
+SELECT * From animals Where (weight_kg >= '10.4' and weight_kg <= '17.3');
+
+/* Second Project*/
+
+--PART ONE - transactions - species column and insertion
+UPDATE animals set species = 'digimon' Where name Like '%mon';
+--PART TWO - transactions - species column and insertion
+UPDATE animals set species = 'pokemon' Where species is null;
+
```

Comment:
- [ ] Please make sure you `verify that change was made` and ``verify that change persists after commit`
```sql
BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 

   SELECT species from animals; 
   COMMIT;
   SELECT species from animals; 
commit
```